### DISCUSSION TITLE
How to run

### DISCUSSION THREAD

**Discussion by zbear2246** [SebLague/Digital-Logic-Sim#99] [Q&A]
How do I run the simulation on Linux? I am on a Linux-compatible Chromebook.

**Comment by Xanderplayz17** [comment]
Sorry, you cannot do that.
There is only a build for linux using the x86-64 arcitecture.
Chromebooks use ARM.

**Comment by Xanderplayz18** [comment]
It has been a while, and this is probably no use to you now, but I compiled it for web at https://xanderplayz16.github.io/DLS-GL/
